k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -2379,7 +2379,7 @@ private void processDraftMessage(LocalMessage message) throws MessagingException
         // Decode the identity header when loading a draft.
         // See buildIdentityHeader(TextBody) for a detailed description of the composition of this blob.
         Map<IdentityField, String> k9identity = new HashMap<IdentityField, String>();
-        if (message.getHeader(K9.IDENTITY_HEADER) != null && message.getHeader(K9.IDENTITY_HEADER).length > 0 && message.getHeader(K9.IDENTITY_HEADER)[0] != null) {
+        if (message.getHeader(K9.IDENTITY_HEADER).length > 0 && message.getHeader(K9.IDENTITY_HEADER)[0] != null) {
```

Comment:
Can you please extract `message.getHeader(K9.IDENTITY_HEADER)` to a local variable.